csp-mixin/signals/azurevirtualnetwork.libsonnet

```diff
@@ -0,0 +1,308 @@
+        },
+      },
+    },
+
+    packetsDropped: {
+      name: 'Packets Dropped DDoS',
+      description: 'Inbound packets per second dropped by DDoS protection',
+      type: 'gauge',
+      sources: {
+        azuremonitor: {
+          expr: 'azure_microsoft_network_virtualnetworks_packetsdroppedddos_countpersecond_maximum{%(queriesSelector)s}',
+          legendCustomTemplate: 'Dropped',
+        },
+      },
+    },
+
+    packetsForwarded: {
+      name: 'Packets Forwarded DDoS',
+      description: 'Inbound packets per second forwarded by DDoS protection',
+      type: 'gauge',
+      sources: {
+        azuremonitor: {
+          expr: 'azure_microsoft_network_virtualnetworks_packetsforwardedddos_countpersecond_maximum{%(queriesSelector)s}',
+          legendCustomTemplate: 'Forwarded',
+        },
+      },
+    },
+    
+    packetsTotal: {
+      name: 'Packets in DDoS',
+      description: 'Total packets per second inbound in DDoS protection.',
+      type: 'gauge',
+      sources: {
+        azuremonitor: {
+          expr: 'azure_microsoft_network_virtualnetworks_packetsinddos_countpersecond_maximum{%(queriesSelector)s}',
+          legendCustomTemplate: 'Total',
+        },
+      },
+    },
+
+    tcpPacketsDropped: {
+      name: 'TCP Packets Dropped DDoS',
+      description: 'Inbound TCP packets per second dropped by DDoS protection',
+      type: 'gauge',
+      sources: {
+        azuremonitor: {
+          expr: 'azure_microsoft_network_virtualnetworks_tcppacketsdroppedddos_countpersecond_maximum{%(queriesSelector)s}',
+          legendCustomTemplate: 'TCP Dropped',
+        },
+      },
+    },
+
+    tcpPacketsForwarded: {
+      name: 'TCP packets Forwarded DDoS',
+      description: 'Inbound TCP packets per second forwarded by DDoS protection',
+      type: 'gauge',
+      sources: {
+        azuremonitor: {
+          expr: 'azure_microsoft_network_virtualnetworks_tcppacketsforwardedddos_countpersecond_maximum{%(queriesSelector)s}',
+          legendCustomTemplate: 'TCP Forwarded',
+        },
+      },
+    },
+    
+    tcpPacketsTotal: {
+      name: 'TCP packets in DDoS',
+      description: 'Total TCP packets per second inbound in DDoS protection.',
+      type: 'gauge',
+      sources: {
+        azuremonitor: {
+          expr: 'azure_microsoft_network_virtualnetworks_tcppacketsinddos_countpersecond_maximum{%(queriesSelector)s}',
+          legendCustomTemplate: 'TCP Total',
+        },
+      },
+    },
+
+    udpPacketsDropped: {
+      name: 'UDP Packets Dropped DDoS',
+      description: 'Inbound UDP packets per second dropped by DDoS protection',
+      type: 'gauge',
+      sources: {
+        azuremonitor: {
+          expr: 'azure_microsoft_network_virtualnetworks_udppacketsdroppedddos_countpersecond_maximum{%(queriesSelector)s}',
+          legendCustomTemplate: 'UDP Dropped',
+        },
+      },
+    },
+
+    udpPacketsForwarded: {
+      name: 'UDP packets Forwarded DDoS',
```

Comment:
TItle or sentence case here?

Reply:
Yeah, good call. I'll clean these all up.. I'm the worst offender for this rule. Pretty sure I'm the reason it exists ;)